mscore/uploadscoredialog.ui

```diff
@@ -18,13 +18,32 @@
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
-    <number>12</number>
+    <number>6</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
+     <item>
+      <widget class="QLabel" name="userAvatarLabel">
+       <property name="minimumSize">
+        <size>
+         <width>48</width>
+         <height>48</height>
+        </size>
+       </property>
+       <property name="maximumSize">
+        <size>
+         <width>48</width>
+         <height>48</height>
+        </size>
+       </property>
+       <property name="text">
+        <string notr="true">User</string>

```

Comment:
and this `notr`?

Reply:
These `notr`s are intentional, these strings are just placeholders which are replaced to actual strings or icons in C++ code.

Reply:
OK. On other such cases an `extracomment` is there to explain this.